docs/compute/manage_software.md

```diff
@@ -0,0 +1,58 @@
+The LEAP JupyterHub environment provides multiple ways to install and manage software. Choose the method that best fits your needs.
+
+##Quick Installs (Temporary)
+
+When you just need software for your current notebook session, you can temporarily install packages using one of the following methods. The lifetime of the installation lasts until the kernel stops. After this, it has to be re-installed.
+
+**1. Conda-Forge**
```

Comment:
The `conda` and `mamba` approaches are pretty similar - do we want to include both? May we want to say why users would choose one over the other?